WDAC-Policy-Wizard/app/Resources/policyTemplates/Recommended_Driver_Blocklist.xml

```diff
@@ -2057,24 +4471,186 @@
+          <FileRuleRef RuleID="ID_DENY_WINKERNEXP_2"/>
+          <FileRuleRef RuleID="ID_DENY_WINKERNEXP_3"/>
+          <FileRuleRef RuleID="ID_DENY_WINKERNEXP_4"/>
+          <FileRuleRef RuleID="ID_DENY_WINRING0_SHA1" />
+          <FileRuleRef RuleID="ID_DENY_WINRING0_SHA256" />
+          <FileRuleRef RuleID="ID_DENY_WINRING0_SHA1_PAGE" />
+          <FileRuleRef RuleID="ID_DENY_WINRING0_SHA256_PAGE" />
+          <FileRuleRef RuleID="ID_DENY_WINRING_1" />
+          <FileRuleRef RuleID="ID_DENY_WINRING_2" />
+          <FileRuleRef RuleID="ID_DENY_WINRING_3" />
+          <FileRuleRef RuleID="ID_DENY_WINRING_4" />
+          <FileRuleRef RuleID="ID_DENY_WINRING_5" />
+          <FileRuleRef RuleID="ID_DENY_WINRING_6" />
+          <FileRuleRef RuleID="ID_DENY_WINRING_7" />
+          <FileRuleRef RuleID="ID_DENY_WINRING_8" />
+          <FileRuleRef RuleID="ID_DENY_WINRING_9" />
+          <FileRuleRef RuleID="ID_DENY_WINRING_10"/>
+          <FileRuleRef RuleID="ID_DENY_WINRING_11"/>
+          <FileRuleRef RuleID="ID_DENY_WINRING_12"/>
+          <FileRuleRef RuleID="ID_DENY_WINRING_13"/>
+          <FileRuleRef RuleID="ID_DENY_WINRING_14"/>
+          <FileRuleRef RuleID="ID_DENY_WINRING_15"/>
+          <FileRuleRef RuleID="ID_DENY_WINRING_16"/>
+          <FileRuleRef RuleID="ID_DENY_WINRING_17"/>
+          <FileRuleRef RuleID="ID_DENY_WINRING_18"/>
+          <FileRuleRef RuleID="ID_DENY_WINRING_19"/>
+          <FileRuleRef RuleID="ID_DENY_WINRING_20"/>
+          <FileRuleRef RuleID="ID_DENY_WINRING_21"/>
+          <FileRuleRef RuleID="ID_DENY_WINRING_22"/>
+          <FileRuleRef RuleID="ID_DENY_WINRING_23"/>
+          <FileRuleRef RuleID="ID_DENY_WINRING_24"/>
+          <FileRuleRef RuleID="ID_DENY_WINRING_25"/>
           <FileRuleRef RuleID="ID_DENY_PROCESSHACKER" />
           <FileRuleRef RuleID="ID_DENY_AMP" />
           <FileRuleRef RuleID="ID_DENY_ASMMAP" />
           <FileRuleRef RuleID="ID_DENY_ASMMAP_64" />
+          <FileRuleRef RuleID="ID_DENY_DBK_32" />
+          <FileRuleRef RuleID="ID_DENY_DBK_64" />
+          <FileRuleRef RuleID="ID_DENY_GDRV" />
           <FileRuleRef RuleID="ID_DENY_PCHUNTER_1" />
           <FileRuleRef RuleID="ID_DENY_PCHUNTER_2" />
           <FileRuleRef RuleID="ID_DENY_PHYMEMX_64" />
-          <FileRuleRef RuleID="ID_DENY_DBK_32" />
-          <FileRuleRef RuleID="ID_DENY_DBK_64" />
-          <FileRuleRef RuleID="ID_DENY_BS_RCIO64_SHA1" />
-          <FileRuleRef RuleID="ID_DENY_BS_RCIO64_SHA256" />
-          <FileRuleRef RuleID="ID_DENY_BS_RCIO64_SHA1_PAGE" />
-          <FileRuleRef RuleID="ID_DENY_BS_RCIO64_SHA256_PAGE" />
         </FileRulesRef>
       </ProductSigners>
     </SigningScenario>
     <SigningScenario Value="12" ID="ID_SIGNINGSCENARIO_WINDOWS" FriendlyName="Auto generated policy on 09-19-2022">
-      <ProductSigners />
+      <ProductSigners>
+        <FileRulesRef>
+            <FileRuleRef RuleID="ID_ALLOW_ALL_2" />
```

Comment:
Allow all references need to be removed. This signing scenario should match the previous version of the blocklist. no updates

Reply:
Removed :)